client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -40,18 +43,22 @@ class PluginAutomatedTransfer extends Component {
 	};
 
 	componentWillMount() {
-		const { COMPLETE, CONFLICTS } = transferStates;
-		const { isTransferring, transferState } = this.props;
+		const { COMPLETE } = transferStates;
+		const {
+			isTransferring,
+			isTransferFailed,
+			transferState,
+		} = this.props;
 
 		if ( COMPLETE === transferState ) {
 			this.setState( { transferComplete: true } );
-		} else if ( isTransferring || CONFLICTS === transferState ) {
+		} else if ( isTransferring || isTransferFailed ) {
```

Comment:
Actually wait wait wait: are you only talking about this particular variable?

Like: replace `isTransferFailed` with `isFailedTransfer`, but leave alone all the other occurrences of "is failed" such as `isAutomatedTransferFailed` and its file `is-automated-transfer-failed.js`